x-pack/plugins/lists/server/scripts/lists/new/list_ip_item.json

```diff
@@ -0,0 +1,4 @@
+{
+  "id": "hand_inserted_item_id",
+  "value": "89.12.89.72"
```

Comment:
I would change that IP if that is a bad one. We don't want to include any internet facing IP's in our source code. Especially if they are "evil" ones ;-)

Reply:
mm yes good call. I can't remember if it was something I found coming off of my local packetbeat or if I pulled it from that bad list but either way I'll replace it with localhost.